content/sse/index.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: SSE and Raw HTTP Streaming API
+section: streaming-api
+index: 0
+api_separator:
+jump_to:
+  Help with:
+    - Authentication#authentication
+    - Connection state recovery#connection-state-recovery
+  API Routes:
+    - Server-sent events#sse
+    - Plain event stream#event-stream
+---
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+HTTP streaming allows for a request from a client to be held by a server, allowing it to push data to the client without further requests. This, much like WebSockets, help avoid the overhead involved in normal HTTP requests. "Server-sent events":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events (SSE) is another method of achieving this for web pages, usually done through the use of "EventSource":https://developer.mozilla.org/en-US/docs/Web/API/EventSource.
+
+This feature is currently in beta. If you're interested in using this feature, or have suggestions on how we can improve it, please "get in touch":https://www.ably.io/contact.
+
+It is subscribe-only: you can not interact with the channel, including to publish, enter presence, query the presence set, attach and detach from channels (without closing and re-opening the stream), or anything else.
+
+Where possible, we strongly recommend the use of one of our "Realtime client libraries":https://www.ably.io/download, which provide "more features and higher reliability":https://support.ably.io/solution/articles/3000061848, and the full use of our "normal realtime messaging API":/realtime.
+
+h2(#authentication). Authentication
+
+The same authentication options as used in the REST API also apply here. So for basic auth, a querystring parameter of @key@ or an @Authorization: Basic <base64-encoded key>@ header; for token auth, a querystring parameter of @accessToken@ containing an access token or an @Authorization: Bearer <base64-encoded token>@ header. See "REST API authentication":/rest-api#authentication for more information.
```

Comment:
> The same authentication options as used in the REST API also apply here

Surely it's the same authentication options used by any clients connecting to Ably over any protocol apply here.  

Then we should link to find out more about Ably authentication, not linking to the raw REST API where customers are going to have to try and figure out how to generate tokens themselves, which is almost never correct.

Reply:
> Surely it's the same authentication options used by any clients connecting to Ably over any protocol apply here. Then we should link to find out more about Ably authentication

I don't follow you here. It's the same underlying protocol as our client libraries use sure, but that's not much help for someone who isn't a client library developer... Our normal authentication docs are written assuming you're using a client library, the only places the actual protocol is documented are the client library development guide, and the raw REST api docs

(Unless you're just saying we should copy & paste (or better transclude) the relevant bits from the rest api docs here, rather than linking?)

> , not linking to the raw REST API where customers are going to have to try and figure out how to generate tokens themselves, which is almost never correct.

Nothing about using the raw REST api means customers have to generate tokens by hand. The [rest api docs](https://www.ably.io/documentation/rest-api#token-authentication) just explain, once you've obtained a token (from a client lib on your auth server or otherwise), how to include it in the REST request you make so that we make (`Authorization` header or querystring param). Which is the the same as what someone using SSE or light-http-streaming will have to do